apps/blestress/src/rx_stress.c

```diff
@@ -837,14 +837,6 @@ rx_stress_10_l2cap_event(struct ble_l2cap_event *event, void *arg)
     MODLOG_DFLT(INFO, "Data buf %s\n", data_buf ? "OK" : "NOK");
     assert(data_buf != NULL);
 
-    /* The first 2 bytes of data is the size of appended pattern data. */
```

Comment:
indeed l2cap uses 2 octects with a size, but maybe this was used for the test purposes?

Reply:
I don't think so - the other board doesn't really read that data, just receives it, detects MTU and calculates bitrate. In theory, this could be used to check the length of the data - but in current state it's done just by using OS_MBUF_PKTLEN().